docs/Using-Fleet/MDM-setup.md

```diff
@@ -53,53 +101,100 @@ Fleet UI:
 
 How to renew the certificate if it's expired or about to expire:
 
-1. Run the `fleetctl generate mdm-apple --email <email> --org <org>` command. Make sure you use the same Apple ID email address that you used when generating the original certificate.
+1. Run the `fleetctl generate mdm-apple --email <email> --org <org>` command. 
+
+2. Sign in to [Apple Push Certificates Portal](https://identity.apple.com) using the same Apple ID you used to get your original certificate. 
 
-2. Sign in to [Apple Push Certificates Portal](https://identity.apple.com) using the same Apple ID you used to get your original certificate. If you don't use the same Apple ID, you will have to turn MDM off and back on for all macOS hosts.
+3. In the **Settings > Integrations > Mobile device management (MDM)** page, under Apple Push Certificates portal, find the serial number of your current certificate. In Apple Push Certificates Portal, click  **Renew** next to the certificate that has the matching serial number. 
 
-3. In the **Settings > Integrations > Mobile device management (MDM)** page, under Apple Push Certificates portal, find the serial number of your current certificate. In Apple Push Certificates Portal, click  **Renew** next to the certificate that has the matching serial number. If you don't renew and get a new certificate, you will have to turn MDM off and back on for all macOS hosts.
+### Renewing SCEP
+TODO
```

Comment:
@noahtalerman is it the case that Fleet will do this automatically in the background?

Reply:
I'm not sure. @mna is this right?

Reply:
To renew SCEP? AFAIK we don't renew it automatically, but I see it gets created with a very long lifetime (10 years), @roperzh might know more about this?

Reply:
that's correct, we don't renew it automatically, however the certificate is created with the validity specified in `mdm.apple_scep_signer_validity_days`, which defaults to 365 (~1 year)

Ideally we would automatically renew this certificate before it expires, but we could optionally change the setting to be something like 10year like Martin initially suspected (other MDMs do this)

Reply:
ah... wait... we're talking about two different SCEP certificates. What I mentioned above is valid for certificates issued to devices for enrollment. Seems like this is for the CA, sorry about that.

As Martin mentioned, we're issuing a certificate that's valid for 10 years when you run `fleetctl generate mdm-apple`. After that, the certificate renewal is not automatic.

Reply:
yes, that's a great summary!